integration/nwo/network.go

```diff
@@ -1517,6 +1552,21 @@ func (n *Network) nextColor() string {
 	return fmt.Sprintf("%dm", color)
 }
 
+// ChaincodeServerRunner returns an ifrit.Runner for the specified chaincode service.
+func (n *Network) ChaincodeServerRunner(c Chaincode, args []string) *ginkgomon.Runner {
+	srvStartCmd := commands.ChaincodeServerStart{PackageID: c.PackageID, ServerArgs: args}
+
+	cmd := NewCommand(c.Path, srvStartCmd)
+
+	return ginkgomon.New(ginkgomon.Config{
+		AnsiColorCode:     n.nextColor(),
+		Name:              c.PackageID,
+		Command:           cmd,
+		StartCheck:        `Starting chaincode .* at .*`,
+		StartCheckTimeout: 15 * time.Second,
+	})
+}
+
```

Comment:
This should not be part of the network; this is part of the test... The network does not launch server chaincode; server chaincode is launched and peer connects to it.